src/Sylius/Bundle/FixturesBundle/Command/FixturesLoadCommand.php

```diff
@@ -31,7 +31,7 @@ protected function configure()
         $this
             ->setName('sylius:fixtures:load')
             ->setDescription('Loads fixtures from given suite')
-            ->addArgument('suite', InputArgument::REQUIRED, 'Suite name')
+            ->addArgument('suite', InputArgument::OPTIONAL, 'Suite name', 'default')
```

Comment:
I'm not sure, fixtures bundle is not aware of suites configuration, but we may actually recommend `default` suite name to end users.